sdks/python/apache_beam/internal/util.py

```diff
@@ -20,9 +20,13 @@
 For internal use only. No backwards compatibility guarantees.
 """
 
+from __future__ import absolute_import
+
 import logging
 import threading
 import weakref
+from builtins import next
```

Comment:
I'd like to understand the reason for adding `from builtins import next`. Is it  done by a conversion tool because we have an occurrence of `next(v_iter)` in the file, and the tool cannot infer whether or not v_iter implements a custom iterator? If yes: will such changes be required  by linter? 

Is my understanding correct that as long as v_iter's class does not implement a custom iterator (and it is not, in our case), the code without the import is still equivalent in Python 2 and Python 3?

Reply:
This is indeed done by the automatic conversion of future. 
And you're right, it isn't necessary here. Lint doesn't seem to complain either, so we can remove this.

Reply:
Hm... the change is harmless, I totally don't mind it. I am a little concerned that we might not be able to enforce consistency in the codebase, and automatically catch when such changes are required. Say, tomorrow somebody adds a usage of `next` in against an object  passed into the method, and we can't tell if the object will has the iterator implemented in Py3-compliant way, so then there would be a difference in behavior between py2 and py3, and importing `next` would make a difference, but if linter does not catch it, chances are reviewers  won't notice it either.

Reply:
I checked again, and it seems like the linter checks for `.next()` calls on objects. So we should be safe.